make-utils.jl

```diff
@@ -1,6 +1,6 @@
 using Base64, Pkg
-Pkg.add(url="https://github.com/TuringLang/TuringTutorials", rev="artifacts")
-using TuringTutorials
+const OUTPUT_DIR = "TuringTutorialsOutput"
+run(`git clone --depth=1 https://github.com/TuringLang/TuringTutorialsOutput $OUTPUT_DIR`)
```

Comment:
Probably better to use https://docs.julialang.org/en/v1/stdlib/LibGit2/#LibGit2.clone so we don't have to rely on an installed `git`:
```suggestion
using LibGit2
const OUTPUT_DIR = "TuringTutorialsOutput"
LibGit2.clone("https://github.com/TuringLang/TuringTutorialsOutput", $OUTPUT_DIR)
```

Reply:
Okay. I've switched to LibGit2. I'm always a bit hesitant with that because it doesn't allow clones with `--depth=1` (https://github.com/libgit2/libgit2/issues/3058). In this case, the difference was 7.2 seconds versus 7.9 seconds, so it isn't too bad.